app/src/main/java/com/cornellappdev/uplift/ui/components/capacityreminder/GymTag.kt

```diff
@@ -0,0 +1,55 @@
+package com.cornellappdev.uplift.ui.components.capacityreminder
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.uplift.util.GRAY04
+import com.cornellappdev.uplift.util.LIGHT_YELLOW
+import com.cornellappdev.uplift.util.PRIMARY_YELLOW
+import com.cornellappdev.uplift.util.montserratFamily
+
+
+@Composable
+fun GymTag(
+    gymName: String,
+    isSelected: Boolean,
+    onSelected: () -> Unit
+) {
+    Box(
+        modifier = Modifier
+            .height(32.dp)
+            .background(
+                color = if (isSelected) LIGHT_YELLOW else Color.White,
+                shape = RoundedCornerShape(8.dp)
+            )
+            .border(
+                width = 1.dp,
+                color = if (isSelected) PRIMARY_YELLOW else GRAY04,
+                shape = RoundedCornerShape(8.dp)
+            )
+            .clickable {
+                onSelected()
+            }, contentAlignment = Alignment.Center
```

Comment:
One nitpick with the visuals! You can see that the click is a little strange looking, forming a solid gray box around the click:

![image](https://github.com/user-attachments/assets/8eaab21d-a03b-44a6-9cb6-c188d4f38d94)

We can likely use `rememberRipple()` (or is it `ripple()`?) OR change this to a Surface (which automatically has ripple) to get this to work. You can try for now passing in an `interactionSource` and `indication` in to `.clickable()` to see if that will make the ripple look correct. If not, perhaps you can switch over to a surface and have that work fine too :-)

Reply:
I added
`.clickable(
                interactionSource = remember { MutableInteractionSource() },
                indication = null
            ) `
to the Box modifier, so hopefully that should fix the gray box on click issue.